gps/doc.go

```diff
@@ -0,0 +1,6 @@
+// Copyright 2018 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package gps is a Go packaging solver library.
```

Comment:
Embedding code examples here could get pretty dense, but maybe this should point to `example.go` at least (especially since it doesn't show up in the go doc as a runnable example).